cas-server-webapp/build.gradle

```diff
@@ -35,7 +35,7 @@ bootRun {
     }
 
     addResources = true
-    jvmArgs = ["-Xrunjdwp:transport=dt_socket,address=5000,server=y,suspend=n"]
+    //jvmArgs = ["-Xrunjdwp:transport=dt_socket,address=5000,server=y,suspend=n"]
```

Comment:
Yeah, this messed up my IDEA debug config, as I run CAS app within IDEA. This is really not necessary. I'll remove it altogether.